lib/rustler_precompiled.ex

```diff
@@ -454,6 +454,13 @@ defmodule RustlerPrecompiled do
 
         %{target_system | arch: arch, vendor: vendor}
 
+      target_system.os =~ "freebsd" ->
+        arch = normalize_linux_arch(target_system.arch)
```

Comment:
I just realized that this may not be specific to Linux :sweat_smile: 
I'm going to change it later.

Reply:
Yeah, I wasn't sure how to name it.

What would you change it to? Create a copy called `normalize_bsd_arch`? Or `normalize_arch` and use it for all of them? Something else?